cfme/tests/webui/test_general_ui.py

```diff
@@ -465,6 +470,48 @@ def test_add_provider_button_accordion(has_no_providers, provider):
     assert view.add_button.is_displayed
 
 
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1642948])
+@pytest.mark.provider([InfraProvider, CloudProvider], selector=ONE_PER_CATEGORY)
+def test_provider_details_page_refresh_after_clear_cookies(
+    appliance, request, setup_provider, provider
+):
+    """
+    Bugzilla:
+        1642948
+    Polarion:
+        assignee: pvala
+        casecomponent: WebUI
+        caseimportance: medium
+        initialEstimate: 1/12h
+        testSteps:
+            1. Navigate to a provider's Details page
+            2. Reboot the appliance
+            3. Click a button or refresh the page or do something on the page and see what happens.
+        expectedResults:
+            1.
+            2.
+            3. You'll be redirected to the Login Page.
+    """
+    view = navigate_to(provider, "Details")
+    appliance.reboot()
+
+    @request.addfinalizer
+    def _finalize():
+        # When the test runs a second time for cloud provider, it raises an error,
+        # this finalizer is workaround for it.
+        navigate_to(appliance.server, "LoggedIn")
```

Comment:
I think `server.logout` is sufficient no need to do navigation on LoggedIn page. what do you think?

Reply:
Hmm, it did the trick.